superbit_lensing/medsmaker/superbit/medsmaker_mocks.py

```diff
@@ -722,7 +736,7 @@ def _meds_metadata(self,magzp=0.0):
         meta['magzp_ref'] = magzp
         return meta
 
-    def _calculate_box_size(self,angular_size,size_multiplier = 2.5, min_size = 16, max_size= 64, pixel_scale = 0.206):
+    def _calculate_box_size(self,angular_size,size_multiplier = 2.5, min_size = 16, max_size= 64, pixel_scale = 0.144):
```

Comment:
This type of thing makes me nervous...I know that the `pixel_scale` shouldn't be changing for superBIT, but what happens if our code gets used by someone else, or for gigabit, etc.? Though there are probably many instances where this assumption is made 🤷‍♂️